src/bind.py

```diff
@@ -44,6 +45,10 @@ class InstallError(SnapError):
     """Exception raised when unable to install dependencies for the service."""
 
 
+class ConfigureError(SnapError):
```

Comment:
Why was ConfigureError added? Will it be handled separately in future code?

Reply:
It is used by configure() but configure() itself is actually a leftover from the parent PR. I have removed both